### PR TITLE
Bedrock Marketplace

### PR DESCRIPTION
**Pull request by pagezyhf**
Blog about bedrock marketplace integration with HF models

### CODE REVIEW

**Comment by pagezyhf** [comment]
I was thinking of merging on Monday  afternoon for more visibility

**Review by jeffboudier — APPROVED** [review]
LGTM!

File: bedrock-marketplace.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+authors:
+- user: pagezyhf
+- user: philschmid
+- user: jeffboudier
+- user: Violette
+---
+
+![logos](/blog/assets/bedrock-marketplace/thumbnail.png)
+
+# How to use Hugging Face models with Amazon Bedrock
```
Comment (Vaibhavs10) [inline bedrock-marketplace.md:13]:
```suggestion
# Use Hugging Face models with Amazon Bedrock
```
More direct

**Review by Vaibhavs10 — APPROVED** [review]
Remember to rebase w/ main before merging, there have been other blogposts that have been merged recently too and update the date ahead of the release.

**Comment by pagezyhf** [comment]
Thank you @Vaibhavs10 and @jeffboudier for the review

**Pull request merged**